src/components/file-upload/file-upload.ts

```diff
@@ -174,11 +174,29 @@ export class MFileUpload extends ModulVue {
 
     private onOpen(): void {
         this.$emit('open');
+        // We need 2 nextTick to be able to have the wrap element in the DOM - MODUL-118
+        Vue.nextTick(() => {
```

Comment:
tu pouvais pas juste wrapper le contenu du dialog dans un div et canceller les drag event la dessus?

Reply:
Je ne crois pas, car le contenu du template du m-file-upload se retrouve dans la slot par défaut qui est dans le body de la fenêtre de dialogue donc la div serait encore là juste dans le body de la fenêtre de dialogue et un wrapper de la fenêtre au complet.

Reply:
Question comme ca: le reste du body ils veulent que les evenements drag fonctionnent ? Pcq on aurait pu trapper sur le body html au complet

Reply:
Ouais, je leur ai demandé et ils veulent que ce soit seulement sur le file-upload ouvert pour le moment. Je voulais moi aussi l'enlever pour l'application au complet...

Reply:
Je pense que ça serait pas plus mal que sur la dragEnter du body il y ai un datatransfer.DragEffect = 'none' comme ça on s'assure que par défaut tu pourras jamais dropper.

Reply:
Tu aurais même pu faire
if (dataTransfer.dropEffect !== 'link') dataTransfer.dragEffect = 'none' je pense pour permettre de linker comme d'habitude.

Reply:
@Mboulianne tu me sembles un expert du drag and drop 🥇  :-)